components/layout/construct.rs

```diff
@@ -1471,6 +1473,7 @@ impl<'a, ConcreteThreadSafeLayoutNode: ThreadSafeLayoutNode>
             ImageUrlOrNone::Url(ref url_value) => {
                 let image_info = Box::new(ImageFragmentInfo::new(
                     url_value.url().map(|u| u.clone()),
+                    node.image_density(),
```

Comment:
I expect this to break because the node won't be an image element. We should just pass None here.